lib/linguist/languages.yml

```diff
@@ -6490,8 +6490,10 @@ Tcl:
   extensions:
   - ".tcl"
   - ".adp"
+  - ".sdc"
   - ".tcl.in"
   - ".tm"
+  - ".xdc"
   filenames:
```

Comment:
I wonder if this should have e.g. before `filenames` a section

```yml
  aliases:
  - sdc
  - xdc
```

If I understand correctly, that is what will enable people to write

\`\`\`sdc
my code snippet
\`\`\`

in GitHub comments, and get them syntax highlighted?

Reply:
@lildude can you confirm if that is how it works?

Reply:
Yes, if you want to use `sdc` and `xdc` to identify the language in codeblocks you will need to add an aliases section.